include/vehicles/tiltrotor/RotorTiltableParams.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef msr_airlib_RotorTiltableParams_hpp
+#define msr_airlib_RotorTiltableParams_hpp
+
+#include "common/Common.hpp"
+#include "vehicles/multirotor/RotorParams.hpp"
+
+namespace msr {
+    namespace airlib {
+
+        struct RotorTiltableParams {
+            
+            bool use_simple_rotor_model = false;  //if true, will use method of Rotor to calculate thrust and torque.
+                                                  //otherwise, will use more complicated model below
+
+            RotorParams rotor_params = RotorParams(); //only some params will be used if use_simple_rotor_model is false
+
+            real_T angle_signal_filter_tc = 0.005f;    //time constant for angle command signal
+            real_T angle_filter_tc = 0.1f;    //time constant for servo arm to reach angle command signal
+
+            //rotor parameters
+            real_T max_voltage = 11.1; //nominal voltage for 3-cell battery. Could increase to 12.6 to indicate battery is fully charged...
+            real_T prop_diameter = 7*(0.0254);
+            real_T motor_resistance = 0.3;
+            real_T air_density = 1.2682;
```

Comment:
I don't think this ever gets used, since `RotorTiltable::updateEnvironmentalFactors()` calls `environment_->getState().air_density` (which interpolates air density based on altitude). They also have a static air density defined in `RotorParams.hpp` as 1.225, but it only gets used in `calculateMaxThrust()` at the bottom of the `RotorParams struct`. Anyway, not a big deal but removing it might clear up any future confusion.

Reply:
Yea, I was originally just using the `air_density` in this file but later decided to get it from `environment_` and forgot to remove it. I'll fix that real quick.